src/smspark/constants.py

```diff
@@ -0,0 +1,5 @@
+
+DRIVER_MEM_INSTANCE_MEM_RATIO = 0.9
```

Comment:
This seems aggressive to me, can we pick a smaller number, say 0.8?

Reply:
This is hard limit, I thinks is fine because:
a) I have tested this with our large datasets
b) think about that for instances with 100G you will waste 10G at 0.9 and 20G at 0.8. 
I don't think it brings any benefit to have it too low, you can still overcommit memory or even swap. I think we should try to go even higher, but for now I feel comfortable at 0.9 but not lower.